### PR TITLE
#10 add slim to the project

### PR DESCRIPTION
**Pull request by Sontsevolosa**
* Adding 'slim-rails' gem to the Gemfile

@mluts Please, review

### CODE REVIEW

**Comment by mluts** [comment]
@Sontsevolosa Please convert existing views to slim. Currently i see only application layout

**Comment by mluts** [comment]
@Sontsevolosa also, include issue number in every commit (e.g.  `Adding 'slim-rails' gem to the Gemfile #10`

**Comment by Sontsevolosa** [comment]
@mluts For converting from html to slim I must use 'html2slim' gem, yes?

**Comment by mluts** [comment]
@Sontsevolosa no, better to do it by hand where it is possible. `html2slim` is more suitable for situations where there are many large html files. Moreover in most cases you should manually fix views after such converters like `html2slim` because they generate not so readable markup code.

**Comment by Sontsevolosa** [comment]
@mluts ok, thanks. I'll read documentation for Slim and make changes in application layout...

**Comment by Sontsevolosa** [comment]
@mluts Please, review my code

File: app/views/layouts/application.html.slim [hunk 1]
```diff
@@ -0,0 +1,10 @@
+doctype html
+html
+  head
+    title Lunchez
+    = stylesheet_link_tag \    'application', media: 'all'
```
Comment (mluts) [inline app/views/layouts/application.html.slim]: @Sontsevolosa why is this `\  ` here ? I suspect that it is not required here...

**Comment by Sontsevolosa** [comment]
@ oh, sorry, you are right.  in my small window 5th line looked like multiple line. I changed it)

**Comment by Sontsevolosa** [comment]
@mluts can I make merge? :)

**Comment by mluts** [comment]
@Sontsevolosa not yet, i want to look to how looks generated html of layout

**Comment by mluts** [comment]
@Sontsevolosa 👍 good to merge

**Pull request merged**